CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing to Headset
+
+Welcome to the Headset community!
+
+## Community
+
+We have two main channels of communication:
+- our Slack workspace: https://tinyurl.com/y7m8y5x4
+- issues on GitHub
+
+If you have any questions on where to start and/or how to proceed, let us know!
+
+## Ways to contribute
+
+We welcome many ways of contributing to Headset:
+- finding bugs and [reporting issues](https://github.com/headsetapp/headset-electron/issues)
+- picking and working on an existing [issue](https://github.com/headsetapp/headset-electron/issues) (the ones tagged
+  "help wanted" are usually a good place to start!)
+- proposing and implementing improvements to Headset
+
+## Setting up the environment
+
+1. Fork the repository on GitHub
+
+2. Clone your fork of the repository
+
+  ```
+  git clone https://github.com/<your-username>/headset-electron.git
+  ```
+
+3. Installation is OS-dependent, so choose the appropriate OS directory:
+  - `windows` for Windows
+  - `darwin` for OSX
+  - `linux` for Linux
+
+    and move to it with `cd <os-directory>`.
+
+4. Install dependencies:
+
+  ```
+  npm install
+  ```
+
+5. Now you can run the app:
+
+  ```
+  npm run production-like
+  ```
+
+You are ready to pick an issue and start working on it.
```

Comment:
Actually not all the issues are possible to work on. Only the ones that marked with a "macOs"/"Windows"/"Linux" tag. the core web app is not open source so it would be kinda confusing.

Reply:
I just added two labels, `client` and `server`. So if people want to help, they can do so on the `client` ones

Reply:
but it's not a server. maybe core/electron ?

Reply:
After some cleanup, only issues marked as "help-wanted" are issues that anybody can actually contribute. All other issues are related to the core of the app which is not open source.
You can use the link https://github.com/headsetapp/headset-electron/labels/help-wanted to point to the relevant issues

Reply:
cool, thanks for clearing that up!